content/docs/faq.md

```diff
@@ -1,147 +1,233 @@
+
+### How can I create my own addon?
+
+Read more about it [here](/docs/develop/getting-started).
+
+### What can I do if I find a problem?
+
+You can tell us using one of these methods.
+
+- Send it through [our feedback form](https://scratchaddons.com/feedback).
+- Create an issue on [the repository](https://github.com/ScratchAddons/ScratchAddons/issues).
+- Create a post on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions).
+- Tell us on [our Discord server](https://discord.gg/R5NBqwMjNc).
+
+### I think you missed a feature. What can I do?
+
+If you think a feature is missing, or you want to suggest an addon to the extension, or you have a good idea, tell us by [following one of the methods mentioned above](#what-can-i-do-if-i-find-a-problem).
+
+### Where can I discuss about Scratch Addons?
+
+You can do it on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions) or [our Discord server](https://discord.gg/R5NBqwMjNc). There, you can discuss about it and ask questions if you're having trouble.
+
+## Technical
+
+### What is an "addon", exactly?
+
+An addon is similar to an extension or a userscript, but they use special APIs provided by the Scratch Addons extension. These APIs allow addons to run scripts on a Scratch page (userscripts), run scripts on the background (persistent scripts), or apply styles to the Scratch website (userstyles).
+
+Userscripts and persistent scripts can use the `addon.*` JavaScript APIs, which allow them to obtain Scratch-related information (for example, get the current logged in user) and also use extension APIs (like sending notifications).
+
+### If everything is an addon, then what does Scratch Addons do?
+
+By itself, Scratch Addons is just an addon loader. Its main tasks are:
+
+- Allow users to enable, disable and configure addons.
+- Run addons and provide APIs to them.
+- Provide useful data to addons (for example, the addon.auth API).
+- Pollute prototypes for use by addon userscripts.
+- Provide ways to access and modify Redux state.
+- Avoid addons from interfering with each other.
+- Avoid duplicate work from different addons.
+
+## Other 
+
+### How can I add/remove myself from the contributors page?
+
+If you want your name to be on the page, please read and follow the instructions of [this issue](https://github.com/ScratchAddons/contributors/issues/{{< specifics/contributors-issue >}}).
+
+If you don’t want your name to be on the page, please tell us by creating an issue on our contributors repository, or by other means of contact. We’re sorry for the inconvenience.
+
+### I have more questions!
+
+If you have more questions that need answers, you can create a post on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions) or send a message [on our Discord server](https://discord.gg/R5NBqwMjNc). Someone will try to answer it for you.
```

Comment:
```suggestion
If you have more questions that need answers, you can create a post on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions) or send a message [on our Discord server](https://discord.gg/R5NBqwMjNc). We will answer as best we can!
```

Reply:
The definition of "we" is quite ambiguous here. The usual users? The developers? Or any other assumption?

I mean "someone" is quite ambiguous, but that's without another thinking of "who created this page".

Reply:
We say the word "our" twice in just the preceding sentence, so "we" refers to "the people that own the linked pages." Trying to refer to "Scratch Addons community" instead sounds awkward, since it's odd that "we" (the people that own the linked pages) aren't responding there. It makes those places sound unofficial.

Reply:
Hmm, good point. I'm still a little bit indecisive but I'll give it a go for now.